beaconChain/helpers/stateTransitionHelpers.ts

```diff
@@ -22,38 +23,72 @@ export function getActiveValidatorIndices(validators: ValidatorRecord[]): int[]
   }, []);
 }
 
+// Modified from: https://github.com/feross/buffer/blob/master/index.js#L1125
+function readUIntBE(array: Uint8Array, offset: number, byteLength: number): number {
```

Comment:
Are we planning on using this elsewhere? If so, perhaps some tests specifically for this would be beneficial

Reply:
We'll probably use this elsewhere?

Since I copied this code from feross/buffer, I'll just copy over the relevant tests, too. :)

I'm assuming this will get pulled into a utility library at some point eventually/soonish. That might be a good time to revisit its testing robustness.

Reply:
Awesome, thank you :) 

I was thinking the same thing, in fact the ssz-js library has some similar functionality, although it interfaces quite differently (https://github.com/ChainSafeSystems/ssz-js/blob/master/src/intBytes.js). Lets keep an eye out for more similarities and migrate when appropriate.